doc/proposals/rlp-target-gateway-resource.md

```diff
@@ -42,22 +42,39 @@ spec:
     kind: HTTPRoute / Gateway
     name: myroute / mygateway
   rateLimits:
-    - operations:
+    - rules:
         - paths: ["/admin/*"]
           methods: ["GET"]
           hosts: ["example.com"]
-      actions:
-        - generic_key:
-            descriptor_key: admin
-            descriptor_value: "yes"
+      configurations:
+        - actions:
```

Comment:
I am wondering what's the advantage for a user to turn this into a `list of actions` instead of just `actions`?

Reply:
To be more aligned with [config.route.v3.RouteAction](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-msg-config-route-v3-routeaction)

It allows you to define more than one descriptor. Not only one. And why that? Because that is more flexible way to define your rate limit. You may want to define one rate limit descriptor for each use case. For example, you may want to define a rate limit descriptor generic and another descriptor depending on some header. If the header does not exist, the second descriptor is not generated, but you still have the other descriptor to rate limit.

Reply:
I see, this is quite helpful then I think having this explanation would help out the user to write better RLPs :)